<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonOpenDirFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Dirrection Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenImageFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDirectionFolder">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>751</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditImageFolder">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>751</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetDirFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>491</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetImageFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>491</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRegexDirectionalityFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.+.txt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRegexImageFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.+.tif</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOut">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>420</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoadVectors">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Vectors</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxImMin">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>-65535.000000000000000</double>
    </property>
    <property name="maximum">
     <double>65535.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>512.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinIntensity">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Intensity</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxIntensity">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>160</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Intensity</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxImMax">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>160</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>605535.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>512.000000000000000</double>
    </property>
    <property name="value">
     <double>2000.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxImageNr">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>200</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelImageNr">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Image number</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxImageScale">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxShowSudoColor">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show sudocolor</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxShapeLineWidth">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>320</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDirectionLineWidth">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>290</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>320</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Shape line width</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxShowSape">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show shape</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Direction line width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Direction line length</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxShowDirection">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show direction</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxDirectionLineLenght">
    <property name="geometry">
     <rect>
      <x>631</x>
      <y>260</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="value">
     <double>20.000000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSameFolders">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>The same folders</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAuloLoadVectors">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Autoload vectors</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>70</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QCheckBox" name="checkBoxShowDirCartesianHistogram">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Show cartesian</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelDirHistBarWidth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Bar Width</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxHistogramBarWidth">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxHistogramScaleCoef">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="prefix">
      <string>*10^</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowPolarHistogram">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Show polar</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelDirectionHistogram">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Direction Histogram</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxHistScale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxHistogramScaleHeight">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowConnected">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show connected</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>250</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSpinBox" name="spinBoxFeatureNr">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelFeatureNr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Feature Nr</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxFeatureHistogramScaleHeight">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxFeatureHistogramScaleCoef">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="prefix">
      <string>*10^</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxFeatureHistogramBarWidth">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowFeatureHistogram">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Show cartesian</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelDirHistBarWidth_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Bar Width</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxHistScale_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
